pydantic/fields.py

```diff
@@ -7,6 +7,7 @@
 import typing
 from copy import copy
 from dataclasses import Field as DataclassField
+from functools import cached_property
```

Comment:
Still need to add checks for import errors (doesn't work in Python 3.7)

Reply:
You could move the import to around line 1001